scripts/release/helpers/terminal.js

```diff
@@ -76,25 +76,11 @@ function capture (cmd, treatStderrAsFailure = true) {
 
   if (flags.debug) {
     log(stdout)
-    if (stderr) {
-      if (flags['no-abort-on-error']) {
-        log(`${RED}${stderr}${RESET}`)
-      } else {
-        fatal(
-          stderr,
-          'Aborting due to error! Use --no-abort-on-error to ignore and continue.'
-        )
-      }
-    }
-  } else if (treatStderrAsFailure && stderr) {
-    if (flags['no-abort-on-error']) {
-      log(`${RED}${stderr}${RESET}`)
-    } else {
-      fatal(
-        stderr,
-        'Aborting due to error! Use --no-abort-on-error to ignore and continue.'
-      )
-    }
```

Comment:
Removing this will unfortunately get us back to the original problem, which prompted me to make this change in the first place: The `branch-diff` tool will exit with a status code zero, but output errors on STDERR if there's a permission issue.

While I have also updated the requirements check to check that the expected permissions are in place, it would still be nice to have some sort of guardrail in place for when `branch-diff` runs into issues.

However, instead of defaulting to throwing on output to STDERR, maybe we should change the default to ignoring it, and only for branch-diff commands we can enable it.

Alternatively, or in combination with this, we could also change the default to only output STDERR, but not abort on it.

What do you think about all this?